src/components/Navbar.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+
+const Navbar = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/about',
+      text: 'About',
+    },
+  ];
+
+  const handleToggle = () => {
+    setNavbarOpen((prev) => !prev);
+  };
+
+  const closeMenu = () => {
+    setNavbarOpen(false);
+  };
+
+  return (
+    <nav className="navBar">
+      <button type="button" onClick={handleToggle}>
+        {navbarOpen ? (
+          <MdClose style={{ color: '#fff', width: '40px', height: '40px' }} />
```

Comment:
Kindly avoid inline styling. It would look better if you use an external CSS to style your component because your code will be hard to maintain and it is not good to write the same code multiple times